platformio_esp82xx_base.ini

```diff
@@ -49,7 +49,7 @@ board_build.f_cpu         = 80000000L
 build_flags               = ${debug_flags.build_flags} ${mqtt_flags.build_flags} -DHTTPCLIENT_1_1_COMPATIBLE=0
 build_unflags             = -DDEBUG_ESP_PORT
                             -fexceptions
-lib_deps                  = td-er/ESPeasySerial @ 2.0.8, I2Cdevlib-Core, Adafruit ILI9341, Adafruit GFX Library, LOLIN_EPD, adafruit/Adafruit BusIO @ ^1.10.0, bblanchon/ArduinoJson @ ^6.17.2, VL53L0X @ 1.3.0, SparkFun VL53L1X 4m Laser Distance Sensor @ 1.2.9, td-er/RABurton ESP8266 Mutex @ ^1.0.2, td-er/SparkFun MAX1704x Fuel Gauge Arduino Library @ ^1.0.1, ESP8266HTTPUpdateServer, FrogmoreScd30, Multi Channel Relay Arduino Library, SparkFun ADXL345 Arduino Library, ITG3205
+lib_deps                  = td-er/ESPeasySerial @ 2.0.8, I2Cdevlib-Core, ESP8266WebServer, Adafruit, Adafruit GFX Library, LOLIN_EPD, adafruit/Adafruit BusIO @ ^1.10.0, bblanchon/ArduinoJson @ ^6.17.2, VL53L0X @ 1.3.0, SparkFun VL53L1X 4m Laser Distance Sensor @ 1.2.9, td-er/RABurton ESP8266 Mutex @ ^1.0.2, td-er/SparkFun MAX1704x Fuel Gauge Arduino Library @ ^1.0.1, ESP8266HTTPUpdateServer, FrogmoreScd30, Multi Channel Relay Arduino Library, SparkFun ADXL345 Arduino Library, ITG3205
```

Comment:
I doubt there is a library called just "Adafruit"

Reply:
That lib is found by its library name 'Adafruit ILI9341', that I pulled local to ESPEasy in this PR to add support for the extra display models. Maybe I should have changed the name, but that is still an option.

Reply:
OK, strange, as it is the worst possible name for a library.